nsqadmin/static/js/app_state.js

```diff
@@ -17,7 +17,11 @@ var AppState = Backbone.Model.extend({
         this.on('change:graph_interval', function(model, v) {
             localStorage.setItem('graph_interval', v);
         });
-        this.set('graph_interval', localStorage.getItem('graph_interval') || 'off');
+
+        var qp = _.object(_.compact(_.map(window.location.search.slice(1).split('&'),
+            function(item) { if (item) { return item.split('='); } })));
+        var interval = qp['t'] || localStorage.getItem('graph_interval') || 'off';
```

Comment:
@jehiah note: there's no real parsing or validation, the code that reads this value requires it to be in the form `/[0-9]+h/`.

Do we need to support other durations?

(I should probably add a little validation here...)